app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "1098951021382",
+    "project_id": "fpvout",
+    "storage_bucket": "fpvout.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:1098951021382:android:0e42e902c98c2d6deb4b47",
+        "android_client_info": {
+          "package_name": "com.fpvout.digiview"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "1098951021382-lge70n5fif1csdumcuk86omabtgvsems.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyDRfdW_nEkshiKZnlUiTY7Ww5R3Ne0Fgog"
```

Comment:
I don't know how this is handled in open-source projects but I'd suggest to not publish the `googles-services.json` since somebody knowing these keys can spam firebase with arbitrary events...

Reply:
good question ...  it is bound to the package. So in the store its not really possible, but I must admit apps not installed from the play store can spam into the account (but why should they do it :-D )

Reply:
After a quick research it turned out to be okay to put the `google-service.json` into the public repository since the keys can be extracted easily from the APK itself. But it is possible to restrict the API access to correctly signed APKs only. 
Take a look at: https://stackoverflow.com/a/57067722/6843341